.golangci.yml

```diff
@@ -117,5 +117,15 @@ issues:
       linters:
         - gosec
 
+    - path: example_test\.go
+      text: "exitAfterDefer"
+      linters:
+        - gocritic
+
+    - path: udp/message/getmid.go
```

Comment:
I think this path is not relevant for v3 branch, so this whole item can be removed.